jsonargparse_tests/test_subcommands.py

```diff
@@ -53,10 +53,20 @@ def test_subcommands_undefined_subcommand(subcommands_parser):
     pytest.raises(ArgumentError, lambda: subcommands_parser.parse_args(["c"]))
 
 
-def test_subcommands_missing_required_subcommand(subcommands_parser):
+def test_subcommands_not_given_when_few_subcommands(subcommands_parser, parser, subparser):
+    err = get_parse_args_stderr(subcommands_parser, [])
+    assert "error: 'expected \"subcommand\" to be one of {a,b,B}, but it was not provided.'" in err
     pytest.raises(ArgumentError, lambda: subcommands_parser.parse_args())
```

Comment:
Don't do other changes. `get_parse_args_stderr(subcommands_parser, [])` is correct. Actually the old test was a mistake. It had to be `subcommands_parser.parse_args([])`. The coverage decreased because of this mistake. This means that one unit test is missing, but it is unrelated to this pull request.

Reply:
okay, I have revised this

Reply:
should I add the missing test in this PR, or open another which can be merged first to ensure the 100% coverage.